### ISSUE TITLE
Harvest for CGIAR

### ISSUE THREAD

**Issue by sbarbosadataverse** [IQSS/dataverse.harvard.edu#62]
ICARDA - https://data.mel.cgiar.org/
CIP - https://data.cipotato.org/dataverse.xhtml
ICRAF - https://data.worldagroforestry.org/

**Comment by djbrooke** [comment]
@jggautier - are you OK taking care of this one?

**Comment by jggautier** [comment]
Yes. I've been testing issues with harvesting lately and this should help. @sbarbosadataverse, I'll get in touch with Leroy from CGIAR for any other details (e.g. which oai-pmh sets to use).

**Comment by jggautier** [comment]
Harvesting is enabled in ICARDA (https://data.mel.cgiar.org/oai?verb=Identify), but seems disabled for CIP (https://data.cipotato.org/oai?verb=Identify) and ICRAF (https://data.worldagroforestry.org/oai?verb=Identify).

I've emailed Leroy to ask if that's correct or are those two repositories using different base urls for OAI-PMH?

**Comment by sbarbosadataverse** [comment]
This is related to indexing issues being worked on my Leonid

**Comment by jggautier** [comment]
https://github.com/IQSS/dataverse.harvard.edu/issues/259 is related.
